packages/web/src/pages/profile-page/ProfilePageProvider.tsx

```diff
@@ -146,9 +141,6 @@ class ProfilePage extends PureComponent<ProfilePageProps, ProfilePageState> {
         getPathname(this.props.location) !== getPathname(location) ||
         action === 'POP'
       ) {
-        this.props.resetProfile()
```

Comment:
So even on web we don't have to reset anymore?

Reply:
yeah cause the "currentUser" will go to that users state, so we don't get weird overlap!

Reply:
Hell yeah, so nice